README.md

```diff
@@ -31,6 +35,40 @@ For more information on Ion have a look at the [Ion Specification](https://amzn.
 * [Ion Java](https://github.com/amzn/ion-java)
 * [Ion Python](https://github.com/amzn/ion-python)
 
+# Master Specification Support
+The current release on master branch  <a title="npm-version" href="https://www.npmjs.com/package/ion-js"><img src="https://img.shields.io/npm/v/ion-js.svg"/></a> 
+is unsupported at this time.
+
+# Development Specification Support
+
+| Types        | IonText | IonBinary | Limitations  |
+|:-------------:|:-------------:|:-------------:|:-------------:|
+| null      | yes | no      | none |
+| bool      | yes      | no      |   none |
+| int | yes      | no      |    underscores, binaryints, bigints |
+| float | yes      | no      |    underscores |
+| decimal | yes      | no      |    none |
+| timestamp | yes      | no      |    offsets/max values non spec-compliant |
+| string | yes      | no      |    values outside of ucs2, escapes between triplequotes |
```

Comment:
What's an example of an "escape between triple quotes"?

Reply:
should be on the github issue